doc/architecture/software-update.rst

```diff
@@ -48,22 +77,55 @@ and applying the updates.
 It can also restore modified files to their pristine state as then were in the
 initial sw image.
 
+.. note::
+The version of swupd client that ships with Ostro 1.0 does not support yet
```

Comment:
almost right... all the "note" text must be indented under the "n" in note:

.. note::
   The version of swupd client...
   signature verification...

The "note" ends with a blank line and text that's no longer indented.